javascript/tagAutocomplete.js

```diff
@@ -90,6 +90,10 @@ const autocompleteCSS = `
         content: "✨";
         margin-right: 2px;
     }
+    .acMetaText span.existed::after {
```

Comment:
Just a name suggestion, existed sounds like it isn't there anymore
```suggestion
    .acMetaText span.used::after {
```